lib/reunion.rb

```diff
@@ -0,0 +1,80 @@
+class Reunion
+
+  attr_reader :name,
+              :activities
+
+  def initialize(name)
+    @name       = name
+    @activities = []
+  end
+
+  def add_activity(activity_object)
+    @activities << activity_object
+  end
+
+  def total_cost
+    sum = 0
+    @activities.each do |activity|
+      activity.participants.each do |name, cost|
+        sum += cost
+      end
+    end
+    sum
+  end
+
+  def breakout
+    total_money_owed = Hash.new(0)
+    @activities.each do |activity|
+      activity.participants.each do |name, cost|
+
+        current_total = total_money_owed[name]
+        total_for_activity = activity.split - cost
+
+        total_money_owed[name] = current_total + total_for_activity
+      end
+    end
+    total_money_owed
+  end
+
+
+  def summary
```

Comment:
This looks good! I don't think this is neatest or cleanest way to get it done, but you got it done! Another approach would be to just add \n onto every person, and then you could use .chomp to remove the final new line or index the string with [0..-2] to remove the final new line.